nixos/modules/services/torrent/opentracker.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.opentracker;
+
+  out = {
```

Comment:
You are right, this pattern is not as widespread in nixpkgs as i have in mind. i will change it to match the general pattern